src/wp-includes/class-wp-comment.php

```diff
@@ -368,7 +368,7 @@ public function __isset( $name ) {
 	 *
 	 * @since 4.4.0
 	 *
-	 * @param string $name
+	 * @param string $name Post name.
```

Comment:
Seems good. 
But in the docblock of `__isset` method, the comment used is `Property name` & not `post name`. see [line 354](https://github.com/WordPress/wordpress-develop/blob/b1d59267affb26777b92f6f2506b149476e8e073/src/wp-includes/class-wp-comment.php#L354).

https://github.com/WordPress/wordpress-develop/blob/b1d59267affb26777b92f6f2506b149476e8e073/src/wp-includes/class-wp-comment.php#L354

Looks like, it'd be consistent if same comment is used for both cases.

Reply:
Indeed, this is a property name, not a post name.